shared/src/hooks/useLatestAPY.ts

```diff
@@ -43,10 +43,18 @@ const getPriceHistoryFromPeriod = (
     return undefined;
   }
 
-  const [periodStartTimestamp, periodEndTimestamp, nextPeriodEndTimestamp] = [
-    periodStart.unix(),
-    periodStart.clone().add(1, "week").unix(),
-    periodStart.clone().add(2, "week").unix(),
+  // add buffers of 5 hours
+  // to catch any early or late settle events
+  const [
+    periodStartTimestamp,
+    periodEndTimestamp,
+    nextPeriodStartTimestamp,
+    nextPeriodEndTimestamp,
+  ] = [
+    periodStart.subtract(5, "hours").unix(),
```

Comment:
we should also do `periodStart.clone()` here, cause `.subtract` will modify the object